static/styles/lib/header.scss

```diff
@@ -0,0 +1,85 @@
+@import "./alerts.scss";
+@import "./colors";
+@import "./bootstrap/scss/bootstrap-flex";
+
+.input-group-addon {
+	width: auto;
+}
+
+.alert-button {
+	.dropdown-menu {
+		left: -20rem !important;
+	}
+
+	@media (max-width: 450px) {
+		.dropdown-menu {
+			width: 70vw;
+			left: -14.5rem !important;
+		}
+
+		.dropdown-menu::before,
+		.dropdown-menu::after {
+			content: none !important;
+		}
+	}
+
+	@media (max-width: 400px) {
+		.dropdown-menu {
+			width: 70vw;
+			left: -14rem !important;
+		}
+
+		.dropdown-menu::before,
+		.dropdown-menu::after {
+			content: none !important;
+		}
+	}
+
+	@media (max-width: 370px) {
+		.dropdown-menu {
+			width: 70vw;
+			left: -13.5rem !important;
+		}
+
+		.dropdown-menu::before,
+		.dropdown-menu::after {
+			content: none !important;
+		}
+	}
+
+	@media (max-width: 300px) {
+		.dropdown-menu {
+			width: 70vw;
+			left: -10rem !important;
+		}
+
+		.dropdown-menu::before,
+		.dropdown-menu::after {
+			content: none !important;
+		}
+	}
+}
+
+.navbar-nav {
+	.minimal-button {
+		float: right;
+		.dropdown-menu {
+			&:after,
+			&:before {
+				left: -320px;
+			}
+
+			&:after {
+				left: 19.75rem;
```

Comment:
It was copied & pasted from other class

Reply:
only values were adjusted to it needs

Reply:
It is the code that was responsible for doing it and here in header the position is just being overriden
			because comparing it to the view from loggedin page the position needed to be changed



```
&:after,
			&:before {
				content: '';
				display: block;
				position: absolute;
				top: -10px;
				width: 0;
				height: 0;
				border-style: solid;
			}

			/* this border color controlls the color of the triangle (what looks like the fill of the triangle) */
			&:after {
				left: 20px;
				border-color: transparent transparent #fff transparent;
				border-width: 11px;
				margin-top: -11px;
			}

			/* this border color controlls the outside, thin border */
			&:before {
				left: 20px;
				border-color: transparent transparent rgba(0, 0, 0, .15) transparent;
				border-width: 11px;
				margin-top: -12px;
			}
```